subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r51/ProjectConfigurationProgressEventCrossVersionSpec.groovy

```diff
@@ -327,10 +321,7 @@ class ProjectConfigurationProgressEventCrossVersionSpec extends ToolingApiSpecif
                 void apply(Project project) {
                     project.configurations.all {
                         if (name == 'foo') {
-                            long start = System.currentTimeMillis()
-                            while (System.currentTimeMillis() - start < $sleepDurationMillis) {
-                                Thread.sleep(${sleepDurationMillis / 2})
-                            }
+                            ${simulateWork(sleepDurationMillis)}
```

Comment:
Also, shouldn't this be `sleepDurationMillis / 2`

Reply:
Please ignore the original `/ 2`. That was a spurious attempt to prevent waiting twice as long as intended.